pkg/controllers/uiplugin/monitoring_test.go

```diff
@@ -252,7 +252,7 @@ func containsPerses(pluginInfo *UIPluginInfo) bool {
 
 var (
 	features       = []string{}
-	clusterVersion = "v4.18"
+	clusterVersion = "v4.19"
```

Comment:
@jgbernalp is it ok to upgrade this for all? Otherwise I can just increase the version for incidents only

Reply:
The tests should probably be refactored to run and expect values for different cluster versions so we guarantee we keep backwards compatibility when needed.

Reply:
refactored the tests: https://github.com/falox/observability-operator/pull/1